### PR TITLE
ilspy: Update to version 6.0.0.5836

### PR DESCRIPTION
**Pull request by omjadas**

### CODE REVIEW

File: bucket/ilspy.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     ],
     "checkver": {
         "github": "https://github.com/icsharpcode/ILSpy",
-        "regex": "v[\\d.]+/ILSpy_binaries_([\\d.]+)\\.zip"
+        "regex": "v(?<tag>[\\d.]+)/ILSpy_binaries_(?<version>[\\d.]+)\\.zip"
     },
     "autoupdate": {
-        "url": "https://github.com/icsharpcode/ILSpy/releases/download/v$matchHead/ILSpy_binaries_$version.zip"
+        "url": "https://github.com/icsharpcode/ILSpy/releases/download/v$matchTag/ILSpy_binaries_$version.zip"
```
Comment (Ash258) [inline bucket/ilspy.json:20]:
```suggestion
        "url": "https://github.com/icsharpcode/ILSpy/releases/download/v$majorVersion.$minorVersion/ILSpy_binaries_$version.zip"
```

File: bucket/ilspy.json [hunk 2]
```diff
@@ -14,9 +14,9 @@
     ],
     "checkver": {
         "github": "https://github.com/icsharpcode/ILSpy",
-        "regex": "v[\\d.]+/ILSpy_binaries_([\\d.]+)\\.zip"
+        "regex": "v(?<tag>[\\d.]+)/ILSpy_binaries_(?<version>[\\d.]+)\\.zip"
```
Comment (Ash258) [inline bucket/ilspy.json:17]:
```suggestion
        "regex": "v[\\d.]+/ILSpy_binaries_([\\d.]+)\\.zip"
```

File: bucket/ilspy.json (hunk 1 above)
Comment (omjadas) [reply bucket/ilspy.json:20]: image


The tag is only major.minor when there is no patch version, when there is a patch version is it major.minor.patch.

Comment (Ash258) [reply bucket/ilspy.json:20]: Then it will be changed for next version

**Pull request merged**